src/pages/todo/components/TodoDetail.js

```diff
@@ -1,12 +1,29 @@
 import React from 'react';
+import { provideHooks, trigger } from 'redial';
+import { connect } from 'react-redux';
+import { DETAIL_TODOS } from '../actions';
+
+@provideHooks({
+  fetch: ({ dispatch, params: { todoId } }) => dispatch({'type': DETAIL_TODOS.REQUEST, id: todoId})
+})
+@connect((state) => {
+  return {
```

Comment:
@vinhkq `... || []` should be `... || null`, since todo is not an array but an object or null.